### PR TITLE
community_feed endpoint implemented.

### PR DESCRIPTION
**Pull request by berkaydemirtas**
community_feed endpoint is created and necessary edge cases are tested. This endpoint can be used to get the posts of the given community in the order of their date.

### CODE REVIEW

File: purbee_backend/backend_source/post/post_type.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ def has_created(self):
     def get_post_type_from_id(post_type_id):
         #this is a db method in database_utilities.py
         post_type_dictionary = get_post_type_from_post_type_id(post_type_id)
+        print(post_type_dictionary)
```
Comment (OnurSefa) [inline purbee_backend/backend_source/post/post_type.py:59]: I encourage you to remove print operations

File: purbee_backend/backend_source/app.py [hunk 2]
```diff
@@ -27,6 +27,24 @@
...
+    if not community :
+        data['response_message'] = "there is no such community."
+        status_code = SC_FORBIDDEN
+        return data, status_code
+
+    data['response_message'] = "community post list successfully returned"
+    data['community_post_list'] = community.post_history_id_list.reverse()
+    status_code = SC_SUCCESS
+    return data, status_code
+
```
Comment (OnurSefa) [inline purbee_backend/backend_source/app.py:46]: I believe some cases are missing: 
* Is there any community with the given <coommunity_id>
* Is the community is public or private, if private does the user have an access to view this community's post history
* Is there any registered user with the given <user_id>

**Review by OnurSefa — CHANGES_REQUESTED** [review]
I have seen some missing points related to the some edge cases in the feed. I have added what cases can be added, also I pointed out the print line which should not be in the our development branch

File: purbee_backend/backend_source/post/post_type.py (hunk 1 above)
Comment (berkaydemirtas) [reply purbee_backend/backend_source/post/post_type.py:59]: I remove this print statement and some other unnecessary print statements.

File: purbee_backend/backend_source/app.py (hunk 2 above)
Comment (berkaydemirtas) [reply purbee_backend/backend_source/app.py:78]: I have implemented this cases. Please check again.

**Review by OnurSefa — APPROVED** [review]

**Comment by OnurSefa** [comment]
I want to point out that, in the app.py one conflict occurs. After you resolved it, this branch can be merged.

**Pull request merged**